src/sass/_disk_device_form.scss

```diff
@@ -6,6 +6,11 @@
   .configuration {
     padding-left: 0;
     width: 8rem;
+
+    p {
```

Comment:
Probably better to target a custom class here, instead of all paragraphs?

Reply:
true, I'm going to use vanilla instead actually